translations/base-ru.yaml

```diff
@@ -440,27 +441,27 @@ buildings:
             description: Многофункциональный - равномерно распределяет все входы на все выходы.
 
         compact:
-            name: Соединитель (компактный)
+            name: Соединитель\n(Компактный)
             description: Объединяет два конвейера в один.
 
         compact-inverse:
-            name: Соединитель (компактный)
+            name: Соединитель\n(Компактный)
             description: Объединяет два конвейера в один.
 
     cutter:
         default:
             name: &cutter Резчик
-            description: Разрезает фигуры сверху вниз и выводит обе половины. <strong>Если вы используете только одну часть, обязательно уничтожьте другую, иначе производство остановится!</strong>
+            description: Разрезает фигуры сверху вниз и выводит обе половины. <strong>Используя только одну часть -  уничтожьте другую, иначе производство остановится!</strong>
         quad:
-            name: Резчик (Четырехпоточный)
-            description: Разрезает фигуры на четыре части. <strong>Если вы используете не все части, обязательно уничтожьте оставшиеся, иначе производство остановится!</strong>
+            name: Резчик\n(4 Выхода)
```

Comment:
This won't work, can we maybe just shorten it so it fits on one line?

Reply:
I'll try to work something out.